src/activities/drawletters/Drawletters.qml

```diff
@@ -0,0 +1,30 @@
+/* GCompris - drawletters.qml
+ *
+ * Copyright (C) 2016 NITISH CHAUHAN <nitish.nc18@gmail.com>
+ *
+ * Authors:
+ *   Olivier Ponchaut <opvg@mailoo.org> (GTK+ version)
```

Comment:
Ok. I'll keep these things in mind.
The image is taken from :- http://gcompris.net/incoming/lang/lang/words/alphabet.png